tools/protoxform/protoprint.py

```diff
@@ -625,6 +688,21 @@ def visit_message(self, msg_proto, type_context, nested_msgs, nested_enums):
                 fields += '%soneof %s {\n%s%s' % (oneof_leading_comment, oneof_proto.name,
                                                   oneof_trailing_comment,
                                                   format_options(oneof_proto.options))
+            # Verify that deprecated fields (that are not hidden and are not of frozen protos)
+            # have a minor version annotation.
+            if field.options.deprecated and not self._frozen_proto and \
```

Comment:
this code/logic seems to be duplicated - i guess moving it to a method will fix that

Reply:
Thanks. It differed a bit, and now if a field is named `DEPRECATED_AND_UNAVAILABLE_DO_NOT_USE` (previously was only relevant for enum values) then it will also be ignored.